### PR TITLE
feat(packages): Add Kafka exporter

### PR DESCRIPTION
**Pull request by EyeCantCU**
#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

### CODE REVIEW

File: kafka_exporter.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package:
+  name: kafka_exporter
+  version: 1.7.0
+  epoch: 1
```
Comment (ajayk) [inline kafka_exporter.yaml:4]:
```suggestion
  epoch: 0
```

Comment (EyeCantCU) [reply kafka_exporter.yaml]: Nice catch. Thank you!

File: kafka_exporter.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      packages: .
+      output: kafka_exporter
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: danielqsj/kafka_exporter
+    strip-prefix: v
```
Comment (ajayk) [inline kafka_exporter.yaml:31]:
```suggestion
    strip-prefix: v

test:
  pipeline:
    - runs: |
        kafka_exporter --version

```

Comment (developer-guy) [reply kafka_exporter.yaml:31]: I added it.

**Review by rawlingsj — APPROVED** [review]

**Pull request merged**